develop/dev-guide-read-in-fast-mode.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Fast Mode
+summary: Introduces a way to speed up the querying of OLAP scenarios by using Fast Mode.
+---
+
+# Fast Mode
+
+> **Warning:**
+>
+> This feature is still an experimental feature. It is NOT recommended that you use it in the production environment.
+
+This document describes how to use Fast Mode to speed up queries in Online Analytical Processing (OLAP) scenarios.
+
+TiFlash supports the following modes:
+
+- `Normal Mode`. The default mode. This mode guarantees the precision of query results and data consistency.
+- `Fast Mode`. This mode does not guarantee the accuracy of query results and data consistency, but provides more efficient query performance.
+
+Some AP scenarios allow for some tolerance to the accuracy of the query results. In these cases, if you need higher query performance, you can switch the corresponding table to TiFlash's Fast Mode for querying.
+
+The mode switch takes effect globally only for those tables whose mode is set to Fast Mode by [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md). TiFlash-related operations are not supported for temporary tables, in-memory tables, system tables, and tables with non-utf-8 characters in the column names, and therefore changing TiFlash table mode is not supported for them.
+
+For more information, see [ALTER TABLE SET TIFLASH MODE](/sql-statements/sql-statement-set-tiflash-mode.md).
+
+## Switch to Fast Mode
+
+By default, all tables are in Normal Mode. You can use the following statement to view the current table mode.
+
+```sql
+SELECT table_mode FROM information_schema.tiflash_replica WHERE table_name = 'table_name' AND table_schema = 'database_name'
+```
+
+Use the following statement to switch the corresponding table to Fast Mode.
+
+{{< copyable "sql" >}}
```

Comment:
seems strange, please check if this is correct.

Reply:
Deleted.